### PR TITLE
Smaller bites reviewed for PR384

### PR DESCRIPTION
**Pull request by dg-latacora**
Smaller portions reviewed for [PR384](https://github.com/crypto101/book/pull/384)
I added some inline feedback on above link where I could contribute

### CODE REVIEW

File: src/exclusive-or.rst [hunk 1]
```diff
@@ -375,28 +373,27 @@ Remaining problems
...
-secure symmetric encryption algorithms aren't the pain point of modern
-cryptosystems. Cryptographers have designed plenty of those, while
-practical key management remains one of the toughest challenges facing
-modern cryptography. One-time pads may solve a problem, but it's the
+that is not the case.
+
+One-time pads are rarely used for being horribly impractical.
+The key is at least as large as all information you would like transmitted,
+*put together*. Plus, the keys must be exchanged securely, ahead
+of time, with all people in your communication network. We like to
```
Comment (dg-latacora) [inline src/exclusive-or.rst:381]: "in your communication network" seems to change meaning for me by getting rid of strangers communicating

Comment (dg-latacora) [inline src/exclusive-or.rst:381]: "in your communication network" seems to change meaning for me by getting rid of strangers communicating

Comment (lvh) [reply src/exclusive-or.rst:381]: Yeah, it's a little different and also I like the original wording better here.

Comment (dg-latacora) [reply src/exclusive-or.rst]: Kept the original sentences with new styles of "would"s

[https://github.com/crypto101/book/pull/408/files#diff-ed836a874730e69e1b9ce4aa4662e0188816fc98d5ad88b977b1174308c6d140R380](https://github.com/crypto101/book/pull/408/files#diff-ed836a874730e69e1b9ce4aa4662e0188816fc98d5ad88b977b1174308c6d140R380)

**Comment by dg-latacora** [comment]
@lvh ,
I'm not sure why the build is failing. Can you help me understand?

**Comment by lvh** [comment]
Hm, looks like at some point the Fedora package for texlive-sourcecodepro stopped actually installing that font. Weird.

**Comment by lvh** [comment]
OK, I can reproduce the issue by rebuilding the Fedora docker image, looks like Fedora broke that image :)

**Comment by lvh** [comment]
Please merge master back into this.

**Comment by dg-latacora** [comment]
I merged master back in and am done with changes

**Comment by lvh** [comment]
Awesome, thanks!

**Pull request merged**